src/pages/Common/LoginPage.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from 'react';
+import { useRecoilValue } from 'recoil';
+import { loggingStateAtom } from 'services/recoil/appState';
```

Comment:
I think this has a typo, should it be logInState? As in the state in the log in process? logging, which comes from the verb, "to log" means:

[transitive verb](https://www.merriam-webster.com/dictionary/transitive)

1a: to cut (trees) for lumber
b: to clear (land) of trees in lumbering —often used with off
2: to make a note or record of : enter details of or about in a log
3a: to move (an indicated distance) or attain (an indicated speed) as noted in a log

Reply:
we are not chopping wood?